setup.py

```diff
@@ -13,13 +13,14 @@
       license='GNU General Public License v3.0',
       python_requires='~=3.6',
       install_requires=[
```

Comment:
One thing that is more for my education. 
Why the list of packages in `install_requires` is shorter than `- pip:` section in `environment.yml`?

Reply:
The list in `install_requires` is sufficient for _installing_ mibilib.

The list in `environment.yml` is for _development_. It therefore also includes packages for testing, generating documentation, etc.

These two options are noted in the readme as Option A and Option B: https://github.com/ionpath/mibilib#option-a-development-clone-repository-and-create-environment

Reply:
I just noted that `setup.py` uses an older version of Python, we should bump that here to get it to match the version in `environment.yml`.

Reply:
Done